pset-iterators.js

```diff
@@ -160,11 +188,15 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.find(function (firstRepublicanMember) {
+  return firstRepublicanMember.party === "Republican"
+})
 
+console.log(firstRepublican.president)
 // Add your code above this line
```

Comment:
@OcielC Great work here using the find method to return the first president of the Republican party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)